Dockerfile

```diff
@@ -11,4 +11,4 @@ COPY package.json package.json
 
 RUN yarn install --silent
 
-CMD ["yarn", "dev"]
\ No newline at end of file
+CMD ["yarn", "start"]
```

Comment:
please, add new line

Reply:
i dont get it :D

Reply:
new line at the end of the file I mean

Reply:
![newline](https://user-images.githubusercontent.com/24206160/55875100-d9e1f200-5b9c-11e9-84a8-e597fd9a2cc9.png)